chapter06/jxp/ch6ex2.pl

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+use feature "say";
+
+say "Please enter some lines";
+chomp( my @lines = <STDIN> );
+
+my %word_count;
+
+foreach my $line ( @lines ){
+  $word_count{$line}++;
+}
+
+foreach my $key ( sort keys %word_count ){
+  if( $word_count{$key} == 1 ){
```

Comment:
Again a nice consideration, maybe this would be a good place to use the single line conditional to decide time/time?